### PR TITLE
Creating verified information section for packages

### PR DESCRIPTION
**Pull request by javanlacerda**
Contribute towards closing #8635

It creates `verified` and `unverified` sections for packages, and moves the `maintainers` and `owners` informations to the verified section.

In the future we intend to implement verification logics for other infos and move that to `verified` section as well.

Example of the package details screen with the new feature:
AsPxhMCRmxVx2bS

cc @di

### CODE REVIEW

File: warehouse/static/sass/blocks/_verified.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  li {
+    list-style-type: circle;
+    list-style-position: inside;
+    padding-left: 20px;
+
+    a {
+        display: inline;
+        padding: 0;
```
Comment (di) [inline warehouse/static/sass/blocks/_verified.scss:33]:
```suggestion
        padding: 0px;
```

File: warehouse/templates/includes/packaging/project-data.html [hunk 2]
```diff
@@ -12,9 +12,44 @@
  # limitations under the License.
 -#}
 
+<div class="sidebar-section verified">
+  <h3 class="sidebar-section__title">{% trans %}Verified details{% endtrans %}</h3>
+  <small><i>These details have been verfied by PyPI</i></small>
```
Comment (ewdurbin) [inline warehouse/templates/includes/packaging/project-data.html:17]:
```suggestion
  <small><i>These details have been verified by PyPI</i></small>
```

File: warehouse/templates/includes/packaging/project-data.html [hunk 3]
```diff
@@ -12,9 +12,44 @@
...
+          </span>
+        </a>
+      </span>
+    {% endfor %}
+  {% endif %}
+</div>
+
+<div class="sidebar-section verified">
+  <h3 class="sidebar-section__title">{% trans %}Unverified details{% endtrans %}</h3>
+  <small><i>These details have <b>not</b> been verfied by PyPI</i></small>
```
Comment (ewdurbin) [inline warehouse/templates/includes/packaging/project-data.html:50]:
```suggestion
  <small><i>These details have <b>not</b> been verified by PyPI</i></small>
```

**Comment by di** [comment]
@ewdurbin There's some details on when we should consider other metadata 'verified' here: https://github.com/pypi/warehouse/issues/8635#issuecomment-2008164863 (below the screenshot)

**Pull request merged**